BlazarBase/src/main/java/com/hubspot/blazar/base/ModuleState.java

```diff
@@ -11,46 +11,46 @@
 
   private final Module module;
   private final Optional<ModuleBuild> lastSuccessfulModuleBuild;
-  private final Optional<RepositoryBuild> lastSuccessfulRepoBuild;
+  private final Optional<RepositoryBuild> lastSuccessfulBranchBuild;
```

Comment:
I agree it would make things a bit more standardized, and would probably make sense to return something like that to the FE. I think though I still need this as an intermediate representation, because of how rosetta creates objects from the mysql results.

Would you prefer something like:
```
{'lastSuccessful': { 
  'moduleBuild': <>, 
  'branchBuild': <>
}}
```